### PR TITLE
I have new edited tur.ini

### PR DESCRIPTION
**Pull request by xAliTura01**
**I have edited new Turkish lang and I want to throw this at you.**
@HighTecDev @jasonwynn10

### CODE REVIEW

File: resources/tur.ini [hunk 1]
```diff
@@ -1,230 +1,311 @@
...
+biome.invalid=§cBu biyom yok
+biome.possible=§eOlası Biyomlar: §f{%0}
+biome.form=Biyom
 
 ; The /p biomes Subcommand
-biomes.name=biomes
-biomes.desc=Biom listesini görüntüler
-biomes.usage=/p biomes
-biomes.possible=Olası biomlar: {%0}
+biomes.name=biome
```
Comment (jasonwynn10) [inline resources/tur.ini:143]: Please leave this as `biomes`. This is separate from the biome command listed above.

**Review by jasonwynn10 — CHANGES_REQUESTED** [review]

File: resources/tur.ini (hunk 1 above)
Comment (xAliTura01) [reply resources/tur.ini:143]: ok sorry

Comment (xAliTura01) [reply resources/tur.ini]: > Please leave this as `biomes`. This is separate from the biome command listed above.

i fixed it.

**Review by jasonwynn10 — CHANGES_REQUESTED** [review]
Please remove color codes from base translations. They should be blank in order for other plugins to add them in as needed

**Comment by xAliTura01** [comment]
> Please remove color codes from base translations. They should be blank in order for other plugins to add them in as needed

oh

> Please remove color codes from base translations. They should be blank in order for other plugins to add them in as needed

i fixed it.

**Review by jasonwynn10 — APPROVED** [review]

**Comment by jasonwynn10** [comment]
Why close this?

**Pull request merged**